### ISSUE TITLE
Larger file size after compression.

### ISSUE THREAD

**Issue by josh08h** [Donaldcwl/browser-image-compression#48]
I'm seeing the file size of a jpeg increase from `1234610` bytes to `2346328` bytes - I've left the 'quality' config option as default.

Could you explain why I am seeing such a large increase in size after compression?

Thanks

**Comment by Donaldcwl** [comment]
@josh08h Would you mind to upload the image file for us to reproduce the issue?

**Comment by josh08h** [comment]
Hi @Donaldcwl 

Image details:
![image](https://user-images.githubusercontent.com/8123886/77529616-dcaa2580-6e87-11ea-9320-d991849fb4eb.png)

Here is the raw image that is uploaded:
![timj-ots0EOYuGtU-unsplash](https://user-images.githubusercontent.com/8123886/77529636-e9c71480-6e87-11ea-99bf-3c321c05bce0.jpg)

```
console.log(origFile.size) // 1234610
const blob = await imageCompression(origFile, { maxWidthOrHeight: 3000 });
console.log(blob.size) // 2346328
```

**Comment by Donaldcwl** [comment]
@josh08h It is fixed in v1.0.9

**Comment by josh08h** [comment]
Perfect.. tested and it works.

Thanks for the quick turnaround @Donaldcwl .
